src/averageAges.js

```diff
@@ -37,7 +41,11 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => withChildren
```

Comment:
Consider breaking down the ternary operator into separate lines for better readability.